tests/app/src/Controller/GetObjectController.php

```diff
@@ -0,0 +1,21 @@
+<?php declare(strict_types=1);
+
+namespace App\Controller;
+
+use Pimcore\Model\DataObject\TestDataObject;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Attribute\AsController;
+use Symfony\Component\HttpKernel\Attribute\Cache;
+
+#[AsController]
+final class GetObjectController
+{
+    #[Cache(smaxage: 3600, public: true)]
```

Comment:
I suppose the lowest tests fail because this attribute is not yet supported in this version. 

Try also adding the annotation (if it already exists in Symfony... maybe we need to install the `sensio/framework-extra-bundle` for the lowest deps instead... or just call the setters on the response 🤔)